### ISSUE TITLE
'PayPalPayIn' object has no attribute 'redirect_url'

### ISSUE THREAD

**Issue by samitnuk** [Mangopay/mangopay2-python-sdk#122]
I updated `mangopaysdk` from v3.1.1  to new release v3.2.0.

`PayPalPayIn` object I can create w/o any problems, but when I want to get `redirect_url` from it I receive:
```
AttributeError: 'PayPalPayIn' object has no attribute 'redirect_url'
```
Here (https://github.com/Mangopay/mangopay2-python-sdk/blob/master/mangopay/resources.py#L453-L459) I can see that in fact class `PayPalPayIn` does not have property `redirect_url`.

Without `redirect_url` I cannot redirect a user to PayPal interface, so payment cannot be made.

If you need some additional information regarding this error, please inform.

**Comment by mickaelpois** [comment]
Hello @samitnuk,

There is an issue that should be shortly fixed, thanks for pointing it.
We will try to provide a quick fix this week (as a 3.2.1 release)

Mickaël

**Comment by mickaelpois** [comment]
This issue will be fix thanks to #123 PR.
A new release will be done this afternoon @samitnuk and @sasha0 !
